### ISSUE TITLE
Insufficient service configuration of "validator.expression", class "ExpressionLanguage" does not exist

### ISSUE THREAD

**Issue by malteschlueter** [symfony/symfony#41468]
**Symfony version(s) affected**: 5.3.0

**Description**
After updating to Symfony 5.3 the container linter outputs the following error.
```
[critical] Error thrown while running command "lint:container". Message: "Invalid service "validator.expression": class "Symfony\Component\ExpressionLanguage\ExpressionLanguage" does not exist."
```
I think it's caused by the changes of #39327.

**How to reproduce**
I will add a example project or from a new Symfony app ASAP. But I would say that it is sufficient to add the Validator component.

**Possible Solution**
Add service `validator.expression_language` only if `ExpressionLanguage` component is installed or add `ExpressionLanguage` component to composer.json of `Validator` component.

**Additional context**
<img width="862" alt="grafik" src="https://user-images.githubusercontent.com/13586874/120221865-677e7100-c23f-11eb-94c3-ff1d4930ebde.png">

**Comment by xabbuh** [comment]
Can you give #41472 a try?

**Comment by gndk** [comment]
@xabbuh I had the same error. #41472 solves it for me.

**Comment by xabbuh** [comment]
@gndk Thanks for the feedback.

**Comment by fbourigault** [comment]
Sorry for the bug 😞

@xabbuh thank you for the fix!

**Comment by malteschlueter** [comment]
@fbourigault No problem^^

Thank you @xabbuh! :)

**Comment by antonmarin** [comment]
Hi. Still receiving with
```
"symfony/console": "^5.3",
"symfony/dependency-injection": "^5.3",
"symfony/error-handler": "^5.3",
"symfony/event-dispatcher": "^5.3",
"symfony/framework-bundle": "^5.3",
"symfony/http-foundation": "^5.3",
"symfony/http-kernel": "^5.3",
"symfony/routing": "^5.3",
"symfony/serializer-pack": "^1.0",
"symfony/validator": "^5.3",
"symfony/yaml": "^5.3"
```
how to solve? is there smth like "symfony/framework-bundle": "5.3.1" ?

**Comment by malteschlueter** [comment]
@antonmarin I think the bugfix was merged one day after releasing `5.3.1`. So you have to wait for `5.3.2` or add `symfony/expression-language` to your composer requirements.

Please correct me if I'm wrong.

**Comment by antonmarin** [comment]
5.3.2 released 👍 Thanks
